src/test/Java/AnagramKataTest.java

```diff
@@ -0,0 +1,53 @@
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+
+public class AnagramKataTest {
+    AnagramKata anagramKata;
```

Comment:
I'd tend to make all fields on a class private by default unless you have a good reason to make them otherwise.